Dockerfile

```diff
@@ -50,6 +29,7 @@ ENV OS Linux
 ENV FS_OVERRIDE 0
```

Comment:
These ENV instructions could be wrapped together in a single ENV (see comment later on)